server/conf/evolutions/default/74.sql

```diff
@@ -0,0 +1,13 @@
+--- Add job_type column
+
+# --- !Ups
+alter table if exists persisted_durable_jobs
+add column if not exists job_type varchar(32) not null
+default 'RECURRING';
+
+alter table if exists persisted_durable_jobs
+alter column job_type DROP DEFAULT;
```

Comment:
question (non-blocking): What does this line do?

Reply:
The first line of the UP adds the new column as not null so you must provide a default value which will be set for all existing rows. The second line removes the default value. It doesn't need to hang around after setting existing rows.

It's cleaner then create the column as nullable, doing an update, and making the column not null.

Reply:
makes sense. thanks for explaining!